lib/rules/value-keyword-case/README.md

```diff
@@ -164,3 +164,64 @@ a {
 }
 ```
 
+### `ignoreProperties: ["/regex/", "non-regex"]`
+
+Ignore case of the values of the listed properties.
+
+For example, with `"lower"`.
+
+```js
+["/^(b|B)ackground$/", "display"]
+```
+
+The following patterns are considered violations:
+
+```css
+a {
+  text-align: LEFT;
+}
+```
+
+```css
+a {
+  text-align: Left;
+}
+```
+
+The following patterns are *not* considered violations:
+
+```css
+a {
+  display: bloCk;
+},
+```
+
+```css
+a {
+  display: BloCk;
+}
+```
+
+```css
+a {
+  display: BLOCK;
+}
+```
+
+```css
+a {
+  display: block;
+}
+```
+
+```css
+  div {
+    background: Red;
+  }
+```
+
+```css
```

Comment:
1. Remove indent, please.
2. Use the `a` selector, as per [these guidelines](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#example-patterns).